.travis.yml

```diff
@@ -2,10 +2,32 @@ language: rust
 sudo: false
 env:
   global:
-    - RUST_BACKTRACE=1
+  - RUST_BACKTRACE=1
 cache: cargo
 
-matrix:
-  include:
-    - rust: nightly
-    - rust: stable
\ No newline at end of file
+# Run builds in Linux, MacOS X and Windows
+os:
+- linux
+- osx
```

Comment:
We found this can *dramatically* slow down test feedback since the queue for Mac machines can get quite long. But it's fine. :)